build/tasks/build.js

```diff
@@ -81,7 +84,7 @@ module.exports = function( grunt ) {
 
 			// Remove define wrappers, closure ends, and empty declarations
 			contents = contents
-				.replace( /define\([^{]*?{/, "" )
+				.replace( /define\([^{]*?{\s*(?:"use strict";)?/, "" )
```

Comment:
Maybe I got carried away with the comment matching, but https://www.npmjs.com/package/standard requires single-quoted strings and prohibits trailing semicolons. Since jquery/contribute.jquery.org/issues/138 proposes adopting it, we should be ready here (especially given that we have no verification that excess "use strict" expressions are absent from the built file).